spec/factories/contract_period_factory.rb

```diff
@@ -10,5 +10,9 @@
     initialize_with do
       ContractPeriod.find_or_create_by(year:)
     end
+
+    trait :consecutive_years do
```

Comment:
why do we need an extra trait? can we not use contract period as the setup is similar without the rest?